### PR TITLE
Auto Fixtures

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Seperates fixture data into those that are automatically updated and those that need manual intervention

It also moves image fixture generation from a manual process to an automated one.

### Before
`/fixtures/...`

### After
`/fixtures/generated/...`
`/fixtures/manual/...`

## Why?
To make it clearer which data should not be touched and also as a way to audit what data could be migrated to being automatically generated

### CODE REVIEW

**Comment by oliverlloyd** [comment]
[Refactor fixtures](https://trello.com/c/ag4isw4x/2675-refactor-fixtures)

File: scripts/test-data/gen-fixtures.js [hunk 1]
```diff
@@ -109,24 +113,54 @@ const HEADER = `/**
...
 try {
+	// Article fixtures
 	const requests = articles.map((article) => {
 		return fetch(`${article.url}.json?dcr`)
 			.then((res) => res.json())
 			.then((json) => {
 				// Add test config
 				json.config = config;
+				// TODO: Remove this when we add in support for CAPI format to DCR
+				delete json.format;
```
Comment (oliverlloyd) [inline scripts/test-data/gen-fixtures.js:124]: This needs to be removed when @buck06191's change is brought in

Comment (buck06191) [reply scripts/test-data/gen-fixtures.js:124]: https://github.com/guardian/dotcom-rendering/pull/2634 is the one to watch

**Review by liywjl — APPROVED** [review]
👍 nice work. No more manually editing fixtures

**Pull request merged**